server/server/src/wsserver.ts

```diff
@@ -40,7 +39,6 @@ export async function startServer (
   serverToken: string,
   options: ServerOptions
 ): Promise<Server> {
-  regNotificationMappers()
```

Comment:
Why they are removed here?

Reply:
I don't know ;) reg calendar mapper moved to , I did the same